macro-plantuml-macro/pom.xml

```diff
@@ -54,7 +54,7 @@
     <dependency>
       <groupId>net.sourceforge.plantuml</groupId>
       <artifactId>plantuml</artifactId>
-      <version>1.2023.9</version>
+      <version>1.2025.2</version>
```

Comment:
For next time, it would be better to create a separate jira issue for different topics and thus separate PRs, as it allows to see that in the release notes.  For now, I'll just create a new issue and link it. Thx